package.json

```diff
@@ -98,7 +98,7 @@
     "cookies": "^0.8.0",
     "debug": "^4.3.4",
     "moment": "^2.29.4",
-    "path-to-regexp": "^6.2.1",
+    "path-to-regexp": "^8.1.0",
     "qs": "^6.11.2",
     "send": "^0.18.0",
```

Comment:
Send also has a high severity warning. Can you also update it to 0.19.0?

Reply:
@steven-sheehy send is not "high severity" warning, just "moderate". I updated the version, but that doesn't help as restify depends on the vulnerable version.

Reply:
What are we waiting at the movement to get this PR merged?